Common/OpenCL/Filters/itkGPUResampleImageFilter.hxx

```diff
@@ -1135,10 +1184,11 @@ GPUResampleImageFilter<TInputImage, TOutputImage, TInterpolatorPrecisionType>::G
  * ***************** PrintSelf ***********************
  */
 
-template <typename TInputImage, typename TOutputImage, typename TInterpolatorPrecisionType>
+template <typename TInputImage, typename TOutputImage, typename TInterpolatorPrecisionType, TTransformPrecisionType>
```

Comment:
Missing typename keyword! Visual C++ says:
> itkGPUResampleImageFilter.hxx(1187,116): error C2061: syntax error: identifier 'TTransformPrecisionType'

Reply:
@ntatsisk  I see, you fixed it, by https://github.com/SuperElastix/elastix/compare/86468c87bef8c29cf7236159cff91d232fb87680..5901d51a30c5a67604a5081287a282e2075c9eff  👍